diff --git a/node_modules/terminal-kit/lib/termconfig/README b/node_modules/terminal-kit/lib/termconfig/README
deleted file mode 100644
index c207592..0000000
--- a/node_modules/terminal-kit/lib/termconfig/README
+++ /dev/null
@@ -1,9 +0,0 @@
-
-
-<terminal>.js refers to the real terminal application, e.g. gnome-terminal, konsole, ...
-
-<terminal>.generic.js refers to a fail-safe version of a terminal used when we got the terminal identifier
-using the $TERM environment variable and it happens that this identifier is shared with not-so-compliant terminal,
-e.g. xterm-256color.
-
-terminfo.js contains for each method the terminfo name of the capability.
diff --git a/node_modules/terminal-kit/lib/termkit.js b/node_modules/terminal-kit/lib/termkit.js
index ca0c4c9..b67df28 100644
--- a/node_modules/terminal-kit/lib/termkit.js
+++ b/node_modules/terminal-kit/lib/termkit.js
@@ -50,7 +50,7 @@ termkit.globalConfig = {} ;
 
 
 
-lazy.requireProperty( termkit , 'tty' , './tty.js' ) ;
+termkit.tty = require( './tty.js' ) ;
 
 // For some reason, starting from node v4, once process.stdin getter is triggered, the 'tty' command would not work properly.
 // This 'hack' cache the result of the command 'tty' if we are in the linux console, so 'gpm' can work.
@@ -70,51 +70,48 @@ if ( process.platform === 'win32' ) { require( './windows.js' )( termkit ) ; }
 
 
 
-// Lazy submodules
-lazy.requireProperties( termkit , {
-	image: './image.js' ,
-	Palette: './Palette.js' ,
-	Rect: './Rect.js' ,
-	ScreenBuffer: './ScreenBuffer.js' ,
-	ScreenBufferHD: './ScreenBufferHD.js' ,
-	TextBuffer: './TextBuffer.js' ,
-	Vte: './vte/Vte.js' ,
-	autoComplete: './autoComplete.js' ,
-	spChars: './spChars.js' ,
-
-	// Document model
-	Element: './document/Element.js' ,
-	Document: './document/Document.js' ,
-	Container: './document/Container.js' ,
-	Text: './document/Text.js' ,
-	AnimatedText: './document/AnimatedText.js' ,
-	Button: './document/Button.js' ,
-	ToggleButton: './document/ToggleButton.js' ,
-	TextBox: './document/TextBox.js' ,
-	EditableTextBox: './document/EditableTextBox.js' ,
-	Slider: './document/Slider.js' ,
-	Bar: './document/Bar.js' ,
-	LabeledInput: './document/LabeledInput.js' ,
-	InlineInput: './document/InlineInput.js' ,
-	InlineFileInput: './document/InlineFileInput.js' ,
-	InlineMenu: './document/InlineMenu.js' ,
-	Inspector: './document/Inspector.js' ,
-	Form: './document/Form.js' ,
-	RowMenu: './document/RowMenu.js' ,
-	ColumnMenu: './document/ColumnMenu.js' ,
-	ColumnMenuMulti: './document/ColumnMenuMulti.js' ,
-	ColumnMenuMixed: './document/ColumnMenuMixed.js' ,
-	SelectList: './document/SelectList.js' ,
-	SelectListMulti: './document/SelectListMulti.js' ,
-	DropDownMenu: './document/DropDownMenu.js' ,
-	TextTable: './document/TextTable.js' ,
-	Layout: './document/Layout.js' ,
-	Border: './document/Border.js' ,
-	Window: './document/Window.js' ,
-
-	// External modules
-	chroma: 'chroma-js'
-} ) ;
+termkit.image = require( './image.js' ) ;
+termkit.Palette = require( './Palette.js' ) ;
+termkit.Rect = require( './Rect.js' ) ;
+termkit.ScreenBuffer = require( './ScreenBuffer.js' ) ;
+termkit.ScreenBufferHD = require( './ScreenBufferHD.js' ) ;
+termkit.TextBuffer = require( './TextBuffer.js' ) ;
+termkit.Vte = require( './vte/Vte.js' ) ;
+termkit.autoComplete = require( './autoComplete.js' ) ;
+termkit.spChars = require( './spChars.js' ) ;
+
+// Document model
+termkit.Element = require( './document/Element.js' ) ;
+termkit.Document = require( './document/Document.js' ) ;
+termkit.Container = require( './document/Container.js' ) ;
+termkit.Text = require( './document/Text.js' ) ;
+termkit.AnimatedText = require( './document/AnimatedText.js' ) ;
+termkit.Button = require( './document/Button.js' ) ;
+termkit.ToggleButton = require( './document/ToggleButton.js' ) ;
+termkit.TextBox = require( './document/TextBox.js' ) ;
+termkit.EditableTextBox = require( './document/EditableTextBox.js' ) ;
+termkit.Slider = require( './document/Slider.js' ) ;
+termkit.Bar = require( './document/Bar.js' ) ;
+termkit.LabeledInput = require( './document/LabeledInput.js' ) ;
+termkit.InlineInput = require( './document/InlineInput.js' ) ;
+termkit.InlineFileInput = require( './document/InlineFileInput.js' ) ;
+termkit.InlineMenu = require( './document/InlineMenu.js' ) ;
+termkit.Inspector = require( './document/Inspector.js' ) ;
+termkit.Form = require( './document/Form.js' ) ;
+termkit.RowMenu = require( './document/RowMenu.js' ) ;
+termkit.ColumnMenu = require( './document/ColumnMenu.js' ) ;
+termkit.ColumnMenuMulti = require( './document/ColumnMenuMulti.js' ) ;
+termkit.ColumnMenuMixed = require( './document/ColumnMenuMixed.js' ) ;
+termkit.SelectList = require( './document/SelectList.js' ) ;
+termkit.SelectListMulti = require( './document/SelectListMulti.js' ) ;
+termkit.DropDownMenu = require( './document/DropDownMenu.js' ) ;
+termkit.TextTable = require( './document/TextTable.js' ) ;
+termkit.Layout = require( './document/Layout.js' ) ;
+termkit.Border = require( './document/Border.js' ) ;
+termkit.Window = require( './document/Window.js' ) ;
+
+// External modules
+termkit.chroma = require( 'chroma-js' ) ;
 
 
 
